CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 -   `PickledInboundGroupSession.sender_signing_key` is now optional.
     ([#89](https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/89))
 
+-   Add `Unknown` to `EncryptionAlgorithm`, representing unsupported algorithms
```

Comment:
Does this count as a breaking change?

Reply:
Kinda. The js-sdk code to handle this is going to be incorrect, for instance, since `Unknown` != `Olm`.

```ts
    rustAlgorithms.forEach((algorithm) => {
        switch (algorithm) {
            case RustSdkCryptoJs.EncryptionAlgorithm.MegolmV1AesSha2:
                algorithms.add("m.megolm.v1.aes-sha2");
                break;
            case RustSdkCryptoJs.EncryptionAlgorithm.OlmV1Curve25519AesSha2:
            default:
                algorithms.add("m.olm.v1.curve25519-aes-sha2");
                break;
        }
    });
```

... But I'm not really sure it's worth a major version bump. I'd treat it as a normal change.

Please put it at the top of the changelog though (most recent changes in the changelog are first, obviously), and resolve the conflict.